sci-biology/fastqc/fastqc-0.11.3.ebuild

```diff
@@ -1,39 +1,45 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
 inherit java-pkg-2 eutils java-ant-2
 
 DESCRIPTION="Quality control FASTA/FASTQ sequence files"
-HOMEPAGE="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
-SRC_URI="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v"${PV}"_source.zip"
+HOMEPAGE="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
+SRC_URI="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v"${PV}"_source.zip"
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS=""
-IUSE=""
+KEYWORDS="~amd64"
 
-DEPEND="sci-biology/picard
-	sci-libs/jhdf5
-	>=virtual/jre-1.5:*"
-RDEPEND="${DEPEND}
+RDEPEND="
 	dev-lang/perl
+	>=virtual/jre-1.5:*
```

Comment:
remove the jdk/jre requirement, the eclasses ensure that.

Reply:
Yes, you are right! Will remove them.

Reply:
When I remove jdk from the dependencies, I am getting
```
QA Notice: Package is using java-ant, but doesn't depend on a Java VM
!!! ERROR: Couldn't find a VM dep
!!! ERROR: Couldn't find a VM dep
 * Could not find valid -source/-target values
```
and the build fails. I guess, an explicit jdk dependency is still necessary.